conf/cassandra.yaml

```diff
@@ -955,14 +957,14 @@ sstable_preemptive_open_interval_in_mb: 50
 # mostly sequential IO when streaming data during bootstrap or repair, which
 # can lead to saturating the network connection and degrading rpc performance.
 # When unset, the default is 200 Mbps or 25 MB/s.
-# stream_throughput_outbound_megabits_per_sec: 200
+# stream_throughput_outbound: 200MiB/s
```

Comment:
I know we're moving away from rate suffixes in this patch, but I'd still suggest that we call this something like `stream_throughput_outbound_rate`, which is a bit more clear (without context) and is, of course, still not unit-specific.